lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -113,14 +113,17 @@ public Thread StartTaskOnNewThread()
                 }
                 catch (TaskStartHandlerException e)
                 {
+                    Logger.Log(Level.Info, "TaskRuntime::TaskStartHandlerException");
                     _currentStatus.SetException(e.InnerException);
                 }
                 catch (TaskStopHandlerException e)
                 {
+                    Logger.Log(Level.Info, "TaskRuntime::TaskStopHandlerException");
                     _currentStatus.SetException(e.InnerException);
                 }
                 catch (Exception e)
                 {
+                    Logger.Log(Level.Info, "TaskRuntime::Exception {0}", e.GetType());
```

Comment:
In this code we set exception which is logged later. This log is more to keep track of which code path set this exception, and for unification with two previous clauses.